cmd/lotus/daemon.go

```diff
@@ -160,6 +169,19 @@ var DaemonCmd = &cli.Command{
 			Name:  "restore-config",
 			Usage: "config file to use when restoring from backup",
 		},
+		&cli.BoolFlag{
```

Comment:
Let's not have this in lotus daemon -- instead, I think this should be a standalone process that connects to a lotus node (exactly the way [the miner does this](https://github.com/filecoin-project/lotus/blob/b7227d3a6b2b9adfde8ab6f2ae04e2cf2bffa481/cmd/lotus-miner/run.go#L77-L81)). 

Do you think that's reasonable? If so, can you please move this into a new lotus-slasher process? If not, happy to discuss more!

Reply:
I have tried using lotus-slasher before, But encountered the problem of rpc blocking(too many rpc request)

Reply:
If there is a good idea to solve this problem, I would be happy to add an independent process

Reply:
Hmm, I'm a little surprised to hear that -- @magik6k do you have any suggestions here?

It's fine to leave it as a goroutine for now, though.